pkg/test/framework/resource/settings.go

```diff
@@ -80,6 +80,10 @@ type Settings struct {
 
 	// Skip VM related parts for all the tests.
 	SkipVM bool
+
+	// Versions are the Istio versions accessible to each of the clusters.
```

Comment:
I think we should clarify this a bit. We should say that each version of Istio will be deployed as indicated by the topology.

Reply:
Added more detail to the comment.